Sprint-1/errors/4.js

```diff
@@ -1,2 +1,7 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+// const 12HourClockTime = "20:53";
+// const 24hourClockTime = "08:53";
+/*
+Variable names must begin with a letter, 
+underscore, non-number character. Each language has its own conventions.*/
+const HourClockTime12 = "20:53";
+const hourClockTime24 = "08:53";
```

Comment:
(Extra question) Have you also noticed the variable names do not quite match the values assigned to the variable?

Reply:
Yes, I see the issue you're pointing out. The variable names suggest a certain format for the time, but the values assigned to them don't match that format. 
so correct way is:
const HourClockTime12 = "08:53";
const hourClockTime24 ="20:53";